src/gui/transferlistwidget.cpp

```diff
@@ -743,6 +752,8 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
     QAction actionCopy_name(GuiIconProvider::instance()->getIcon("edit-copy"), tr("Copy name"), 0);
     connect(&actionCopy_name, SIGNAL(triggered()), this, SLOT(copySelectedNames()));
+    QAction actionCopy_hash(GuiIconProvider::instance()->getIcon("edit-copy"), tr("Copy hash"), 0);
+    connect(&actionCopy_hash, SIGNAL(triggered()), this, SLOT(copySelectedHashes()));
```

Comment:
`connect(&actionCopy_hash, &QAction::triggered, this, &TransferListWidget::copySelectedHashes);`